src/main/java/core/basesyntax/dataservices/DataConverterImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.dataservices;
+
+import core.basesyntax.FruitTransaction;
```

Comment:
The import statement for `FruitTransaction` should be from a `model` package if it exists, to keep the structure of the code organized.